infrastructure/joke_notifier.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_lambda_function" "joke_notifier" {
+  function_name = "joke_notifier"
+  filename        = "build/sns-notifier.zip"
+  role          = local.iam_role
+  handler       = "sns_notifier.handler"
+  timeout       = 300
+  runtime       = "python3.9"
+
+environment {
+    variables = {
+      JOKES_TOPIC_ARN = aws_sns_topic.jokes_topic.arn
```

Comment:
you call it here "JOKES_TOPIC_ARN" but in the function you name it 

"TOPIC_ARN"
They need to be the same